sample_consensus/include/pcl/sample_consensus/model_types.h

```diff
@@ -68,7 +68,9 @@ namespace pcl
   };
 }
 
-// Define the number of samples in SacModel needs
+// Warning: this array is deprecated and is kept only to prevent breaking
```

Comment:
Hm, actually yes. For some reason I thought that depreciation attribute can not be applied to variables.